starkinfra/issuingtoken/__issuingtoken.py

```diff
@@ -0,0 +1,239 @@
+    - device_imei [string]: device imei used for tokenization. ex: "352099001761481"
+    - wallet_instance_id [string]: unique id refered to the wallet app in the current device. ex: "71583be4777eb89aaf0345eebeb82594f096615ed17862d0"
+    """
+
+    def __init__(self, card_id=None, wallet_id=None, wallet_name=None, merchant_id=None, id=None,
+                created=None, updated=None, external_id=None, status=None, tags=None, activation_code=None,
+                method_code=None, device_type=None, device_name=None, device_serial_number=None, device_os_name=None,
+                device_os_version=None, device_imei=None, wallet_instance_id=None):
+        Resource.__init__(self, id=id)
+
+        self.card_id = card_id
+        self.wallet_id = wallet_id
+        self.wallet_name = wallet_name
+        self.merchant_id = merchant_id
+        self.created = created
+        self.updated = updated
+        self.external_id = external_id
+        self.status = status
+        self.tags = tags
+        self.activation_code = activation_code
+        self.method_code = method_code
+        self.device_type = device_type
+        self.device_name = device_name
+        self.device_serial_number = device_serial_number
+        self.device_os_name = device_os_name
+        self.device_os_version = device_os_version
+        self.device_imei = device_imei
+        self.wallet_instance_id = wallet_instance_id
+
+
+_resource = {"class": IssuingToken, "name": "IssuingToken"}
+
+
+def get(id, user=None):
+    """# Retrieve a specific IssuingToken
+    Receive a single IssuingToken object previously created in the Stark Infra API by its id
+    ## Parameters (required):
+    - id [string]: object unique id. ex: "5656565656565656"
+    ## Parameters (optional):
+    - user [Organization/Project object, default None]: Organization or Project object. Not necessary if starkinfra.user was set before function call
+    ## Return:
+    - IssuingToken object with updated attributes
+    """
+    return rest.get_id(resource=_resource, id=id, user=user)
+
+
+def query(limit=None, after=None, before=None, status=None, card_ids=None, tags=None, ids=None, user=None, external_ids=None):
+    """# Retrieve IssuingTokens
+    Receive a generator of IssuingToken objects previously created in the Stark Infra API
+    ## Parameters (optional):
+    - limit [integer, default None]: maximum number of objects to be retrieved. Max = 100. ex: 35
+    - after [datetime.date or string, default None]: date filter for objects created only after specified date. ex: datetime.date(2020, 3, 10)
+    - before [datetime.date or string, default None]: date filter for objects created only before specified date. ex: datetime.date(2020, 3, 10)
+    - status [string, default None]: current IssuingToken status. ex: "active", "blocked", "canceled", "frozen" or "pending"
+    - card_ids [list of strings, default None]: card IDs. ex: ["5656565656565656", "4545454545454545"]
```

Comment:
list of card_ids to filter retrieved objects

Mesma coisa no page()

Reply:
alinhar pra ver se precisa colocar na docstring da classe